docs/production-deployment/cloud/worker-health.mdx

```diff
@@ -327,7 +328,9 @@ The work is not getting processed as fast as it should/can.
 
 Increase the `maxConcurrentWorkflowTaskExecutionSize` and `maxConcurrentActivityExecutionSize` values and keep an eye on your Worker resource metrics (CPU utilization, etc) to make sure you haven't created a new issue.
 
-### Configure Sticky Cache {#configure-sticky-cache}
+### Configure Sticky Execution Cache {#configure-sticky-cache}
+
+A Sticky Execution is a when a Worker Entity caches the Workflow Execution Event History and creates a dedicated Task Queue to listen on.
```

Comment:
```suggestion
Sticky Execution means that a Worker caches a Workflow Execution Event History and creates a dedicated Task Queue to listen on.
It significantly improves performance because the Temporal Service only sends new events to the Worker instead of entire Event Histories.
```